### PR TITLE
Fix/smoke add condition for select drive function

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java [hunk 1]
```diff
@@ -57,8 +57,8 @@ public void goToSpecificMainFunctions(mainEcmFunctions main) {
     switch (main) {
     case EXPLORER:
       info("Select Explorer tab");
-
-      ELEMENT_EXPLORER_CATEGORIES_ECM_FUNCTIONS.click();
+      if($(ELEMENT_ECMS_FUNCTIONS_DRIVES).is(Condition.not(Condition.visible)))
+      {ELEMENT_EXPLORER_CATEGORIES_ECM_FUNCTIONS.click();}
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java]: Indentation is not good.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SpaceActivitiesTestIT.java [hunk 2]
```diff
@@ -105,7 +105,7 @@ public void test03_DeleteYourActivityOnSpace() {
     activityStream.addActivity(activity1, "");
     String id = $(byClassName("activityStream")).parent().getAttribute("id").split("UIActivityLoader")[1];
     // hover on the activity to appear the delete button
-    $(byId(ELEMENT_CONTAINER_ACTIVITY.replace("{id}", id))).find(byClassName(ELEMENT_DATE_ACTIVITY)).hover();
+    $(byId(ELEMENT_CONTAINER_ACTIVITY.replace("{id}", id))).find(byClassName(ELEMENT_DATE_ACTIVITY)).hover().click();
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SpaceActivitiesTestIT.java]: hover() seems useless now, no ? Also we should update the comment.

File: social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java [hunk 3]
```diff
@@ -69,7 +69,7 @@ public void test01_LikeActivity() {
 
         ELEMENT_WHO_LIKED_POPUP.waitUntil(Condition.appears, Configuration.timeout);
         // hover on the activity to appear the delete button
-        $(byId(ELEMENT_CONTAINER_ACTIVITY.replace("{id}", id))).find(byClassName(ELEMENT_DATE_ACTIVITY)).hover();
+        $(byId(ELEMENT_CONTAINER_ACTIVITY.replace("{id}", id))).find(byClassName(ELEMENT_DATE_ACTIVITY)).hover().click();
```
Comment (thomasdelhomenie) [inline social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java]: hover() seems useless now, no ? Also we should update the comment.

File: social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java [hunk 4]
```diff
@@ -106,7 +106,7 @@ public void test02AddComment() {
         $(byXpath(ELEMENT_COMMENT_BUTTON.replace("{id}", id))).pressEnter().waitUntil(Condition.disappears, Configuration.timeout);
         $(byText(comment)).should(Condition.exist);
         // hover on the activity to appear the delete button
-        $(byId(ELEMENT_CONTAINER_ACTIVITY.replace("{id}", id))).find(byClassName(ELEMENT_DATE_ACTIVITY)).hover();
+        $(byId(ELEMENT_CONTAINER_ACTIVITY.replace("{id}", id))).find(byClassName(ELEMENT_DATE_ACTIVITY)).hover().click();
```
Comment (thomasdelhomenie) [inline social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java]: hover() seems useless now, no ? Also we should update the comment.

File: social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java [hunk 5]
```diff
@@ -157,7 +157,7 @@ public void test_15DeleteComment() {
         // verify that the comment is deleted
         $(byText(comment)).shouldNot(Condition.exist);
         // hover on the activity to appear the delete button
-        $(byId(ELEMENT_CONTAINER_ACTIVITY.replace("{id}", id))).find(byClassName(ELEMENT_DATE_ACTIVITY)).hover();
+        $(byId(ELEMENT_CONTAINER_ACTIVITY.replace("{id}", id))).find(byClassName(ELEMENT_DATE_ACTIVITY)).hover().click();
```
Comment (thomasdelhomenie) [inline social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java]: hover() seems useless now, no ? Also we should update the comment.

File: social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java [hunk 6]
```diff
@@ -187,7 +187,7 @@ public void test03_DeleteYourActivity() {
         activityStream.addActivity(activity1, "");
         String id = $(byClassName("activityStream")).parent().getAttribute("id").split("UIActivityLoader")[1];
         // hover on the activity to appear the delete button
-        $(byId(ELEMENT_CONTAINER_ACTIVITY.replace("{id}", id))).find(byClassName(ELEMENT_DATE_ACTIVITY)).hover();
+        $(byId(ELEMENT_CONTAINER_ACTIVITY.replace("{id}", id))).find(byClassName(ELEMENT_DATE_ACTIVITY)).hover().click();
```
Comment (thomasdelhomenie) [inline social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java]: hover() seems useless now, no ? Also we should update the comment.

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java (hunk 1 above)
Comment (ilyeskallala) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java]: reformat code done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SpaceActivitiesTestIT.java (hunk 2 above)
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SpaceActivitiesTestIT.java]: hover() removed and comment was updated

File: social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java (hunk 3 above)
Comment (ilyeskallala) [inline social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java]: hover() removed and comment was updated

File: social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java (hunk 4 above)
Comment (ilyeskallala) [inline social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java]: hover() removed and comment was updated

File: social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java (hunk 5 above)
Comment (ilyeskallala) [inline social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java]: hover() removed and comment was updated

File: social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java (hunk 6 above)
Comment (ilyeskallala) [inline social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOCHomePageTestIT.java]: hover() removed and comment was updated

**Pull request merged**